src/pages/publication/contributors_tab/components/AffiliationsCell.tsx

```diff
@@ -127,4 +119,10 @@ const AffiliationsCell: FC<AffiliationsCellProps> = ({ affiliations, baseFieldNa
   );
 };
 
+const AffiliationElement: FC<any> = ({ unitUri }) => {
+  const [unit, isLoadingUnitHierarchy] = useFetchUnitHierarchy(unitUri);
+
+  return <div>{isLoadingUnitHierarchy ? <Progress /> : unit ? <AffiliationHierarchy unit={unit} /> : null}</div>;
```

Comment:
plain div? eller fragment? eller skal det være noe styling?

Reply:
faktisk med hensyn. Ligger en flex utenfor her, som vil ha to elementer: units til venstre, og fjern-knapp til høyre. Alle nivå av units kommer som egen <p>-tag, så må samle de med en div for at di ikke skal listes horisontalt